packages/module/src/CodeModal/CodeModal.scss

```diff
@@ -80,6 +80,42 @@
   }
 }
 
+// ============================================================================
+// Chatbot Display Mode - Fullscreen
+// ============================================================================
+.pf-chatbot__code-modal--fullscreen {
+  inset-block-end: 0;
+  inset-inline-end: 0;
+  width: 50%;
+  height: fit-content;
+  top: 50%;
```

Comment:
Do these `top` and `left` rules override the `inset-block-end` and `inset-inline-end` rules? I think they both are position rules...

Reply:
They do, but we need them to center the modal since it's basically outside of the usual Bullseye it usually lives in. I'm hoping that we can drop these once we're able to apply classes to the Backdrop, which holds the Bullseye.